modules/network-observability-updating-migrating.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/installing-operators.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-updating-migrating_{context}"]
+= Migrating removed stored versions of the CRD
+
+Before updating to the Network Observability Operator version 1.6, you need take steps to migrate the removed, stored version of the `FlowCollector` CRD. 
```

Comment:
I would introduce it like that:

Network Observability Operator version 1.6 removes the old and deprecated `v1alpha1` version of the `FlowCollector` API. If you ever used it on your cluster, it might still be referenced in the `storedVersion` of the CRD, which is going to block the upgrade process.

To know if you are in this situation, run this command:

```bash
oc get crd flowcollectors.flows.netobserv.io -ojsonpath='{.status.storedVersions}'
```

If `v1alpha1` appears in the list of results, you will need to take additional steps before upgrading.

(and then continue with the procedure)

Reply:
I reworked this a little to add the diagnostic step in the procedure itself. Let me know what you think. I also added the mention to the Storage Version Migrator Operator.